tasks/Makefile.package

```diff
@@ -20,6 +20,8 @@ export REPO_URL ?= https://github.com/$(VENDOR)/$(NAME)
 # $(call github_download_binary_release,version,repo,asset)
 download_binary = $(CURL) -o $(INSTALL_PATH)/$(NAME) $(DOWNLOAD_URL) && chmod +x $(INSTALL_PATH)/$(NAME)
 
+download_tar = $(CURL) -o - $(DOWNLOAD_URL) | tar -zxO $(NAME) > $(INSTALL_PATH)/$(NAME) && chmod +x $(INSTALL_PATH)/$(NAME)
```

Comment:
let's call it `download_tarball`. `tar` is the command and `tarball` is the artifact.

> In computing, tar is a computer software utility for collecting many files into one archive file, often referred to as a tarball,